File: sorting/bubble-sort/javascript/bubble.js
let array = [2, 9, 4, 6, 0];
const n = array.length;

for (let i = 0; i < n; i++) {
  for (let j = 0; j < n; j++) {
    if (array[j] > array[j + 1]) {
      var temp = array[j];
      array[j] = array[j + 1];
      array[j + 1] = temp;
    }
  }
}

for (let i = 0; i < n; i++) {
  console.log(array[i]);
}

File: sorting/bubble-sort/C++/bubble.cpp
#include <bits/stdc++.h>

int main(){
    int arr[]={1,3,4,9,5};
    int n=sizeof(arr)/sizeof(arr[0]);
    int temp;
    
    for(int i=0;i<n;i++){
        for(int j=0;j<n-i;j++){
            if(arr[j]>arr[j+1]){
            //   temp=a[j];
            //  a[j]=a[j+1];
            //  a[j+1]=temp;
            std::swap(arr[j],arr[j+1]);
            }
            
        }   
    }
    
    for(int i=0; i<n;i++){
        std::cout << arr[i] << " ";
    }
}